addons/binding/org.openhab.binding.wink/ESH-INF/thing/thermostat.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
I have to go back to my pull request to remember what it was exactly I was supposed to do, but I think that the maintainers are asking me to move all this xml stuff into annotations on the handler classes.  I'll take a look when I get a chance and update you.

Reply:
I looked at the pull request and I think the xml stuff he was referring to was the files in the OSGI-INF directory and not the xml files used by Paper UI for config.  I could be wrong though.